man/Tss2_Tcti_Socket_Init.3.in

```diff
@@ -1,39 +1,31 @@
 .\" Process this file with
 .\" groff -man -Tascii foo.1
 .\"
-.TH InitSocketTcti 3 "JUNE 2017" Intel "TPM2 Software Stack"
+.TH Tss2_Tcti_Socket_Init 3 "JUNE 2017" Intel "TPM2 Software Stack"
 .SH NAME
-InitSocketTcti \- Initialization function for the Microsoft TPM simulator TCTI library.
+Tss2_Tcti_Socket_Init \- Initialization function for the Microsoft TPM simulator TCTI library.
 .SH SYNOPSIS
-.B #include <tcti/tcti_socket.h>
+.B #include <tcti/tss2_tcti_mssim.h>
```

Comment:
I assume that you have already queued the change from tcti/tcti_socket.h -> tcti/tss2_tcti_mssim.h

Reply:
whoops, didn't mean for that one to sneak in there. I've got the socket -> mssim rename queued up to go in after this. I'll flip this back till then. My mistake.